tests/Build/Changelog/ChangelogBuilderTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Aws\Test\Build\Changelog;
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+class ChangelogBuilderTest extends \PHPUnit_Framework_TestCase
+{
+	/**
+	 * @covers ChangelogBuilder
+	 *
+	 */
+
+	private $RESOURCE_DIR= "tests/Build/Changelog/resources";
```

Comment:
This test shouldn't occur within the SDK's package on disk. (The test runner might not have permission to write to this directory on all systems.) 

Use [`sys-get-temp-dir`](http://php.net/manual/en/function.sys-get-temp-dir.php) to get the system's temporary directory and then create a subdirectory therein.